examples/hf_serialization.py

```diff
@@ -28,6 +28,28 @@
 
 from tensorizer import TensorDeserializer, TensorSerializer, stream_io, utils
 
+s3_access_key_id = os.environ.get("AWS_ACCESS_KEY_ID")
+s3_secret_access_key = os.environ.get("AWS_SECRET_ACCESS_KEY")
+
+if os.environ.get("AWS_ENDPOINT_URL"):
+    default_s3_read_endpoint = os.environ.get("AWS_ENDPOINT_URL")
+    default_s3_write_endpoint = os.environ.get("AWS_ENDPOINT_URL")
+else:
+    default_s3_read_endpoint = "accel-object.ord1.coreweave.com"
+    default_s3_write_endpoint = "object.ord1.coreweave.com"
```

Comment:
1. We aren't AWS
2. Use `or None` to map blank/empty (but still set) environment variables to `None` instead of `""`, since those should be treated equivalently (but `""` has special meaning as an access key to `open_stream`)
3. Setting the endpoint to `None` should already default to the `*object.ord1.coreweave.com` endpoints (or `.s3cfg` if available) through the logic in `open_stream`
```suggestion
s3_access_key_id = os.environ.get("S3_ACCESS_KEY_ID") or None
s3_secret_access_key = os.environ.get("S3_SECRET_ACCESS_KEY") or None
default_s3_write_endpoint  = default_s3_read_endpoint = (
    os.environ.get("S3_ENDPOINT_URL") or None
)
```

Reply:
Points taken. Will commit this.